controllers/user.controller.js

```diff
@@ -124,6 +132,7 @@ const deleteFavoriteBuilding = asyncCatcher(async (req, res, next) => {
   res.json({
     ok: true,
     status: 200,
+    user: user,
   });
```

Comment:
```suggestion
  res.json({
    ok: true,
    status: 200,
    buildingId,
  });
```

데이터를 delete한 경우에 리스폰스에 delete한 데이터가 뭔지 보여주는게 나을거 같은데 어떻게 생각하시나요?

Reply:
delete가 어떤 것이 되었는지 확인 가능하니 더 좋은 것 같네요.

Reply:
수정하겠습니다.